Branch-SDK/Branch-SDK/BNCCrashlyticsWrapper.m

```diff
@@ -0,0 +1,71 @@
+//
+//  BNCCrashlyticsReportingHelper.m
+//  Branch.framework
+//
+//  Created by Jimmy Dee on 7/18/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+#import "BNCCrashlyticsWrapper.h"
+
+@interface BNCCrashlyticsWrapper()
+@property (nonatomic, nullable) id crashlytics;
+@end
+
+@implementation BNCCrashlyticsWrapper
+
++ (id)sharedInstance
```

Comment:
The standard formatter is `clang-format`.  The standard linter is Xcode > Product > Analyze.

The build-bot runs `Xcode Analyze` on every Github PR check in.  I can also add a script to format the code on every checkin, but that would just infuriate everyone.